### PR TITLE
refactor: config tests

### PR DESCRIPTION
**Pull request by rishabh3112**
**What kind of change does this PR introduce?**
refactor

**Did you add tests for your changes?**
Yup

**If relevant, did you update the documentation?**
Not needed,

**Summary**
There are various grammatical mistakes as well as non standard descriptions.
Some cases also have redundant files.

**Does this PR introduce a breaking change?**
Nope

**Other information**
Discussed in #2930

### CODE REVIEW

**Comment by rishabh3112** [comment]
WIP

**Review by alexander-akait — APPROVED** [review]

File: test/build/config/absent/webpack.config-absent.js [hunk 1]
```diff
@@ -1,9 +0,0 @@
-const { resolve } = require("path");
```
Comment (alexander-akait) [inline test/build/config/absent/webpack.config-absent.js:1]: Duplicate?

Comment (rishabh3112) [reply test/build/config/absent/webpack.config-absent.js:1]: Redundant.

**Review by alexander-akait — COMMENTED** [review]
Ready to merge?

**Comment by rishabh3112** [comment]
> Ready to merge?

Still there are a lot of tests with bad descriptions 😞. If this is needs to be merged now, I can open a separate pull request later to address them.

**Comment by alexander-akait** [comment]
You can continue work here :+1:

**Review by alexander-akait — APPROVED** [review]

**Review by snitin315 — APPROVED** [review]

**Pull request merged**